Tests/SendConsignments/SendDigitalStampTest.php

```diff
@@ -57,9 +57,11 @@ public function testSendOneConsignment()
 
             $this->assertEquals(true, preg_match("#^https://api.myparcel.nl/pdfs#", $myParcelCollection->getLinkOfLabels()), 'Can\'t get link of PDF');
 
-            echo "\033[32mGenerated digital stamp shipment: \033[0m";
-            print_r($myParcelCollection->getLinkOfLabels());
-            echo "\n\033[0m";
+            if (!getenv('CI')) {
```

Comment:
```suggestion
            if (! getenv('CI')) {
```

Reply:
Misschien in een trait gooien. Dan aanroepen met `$this->debugLinkOfLabels()` met `message` als parameter

Reply:
Done